### DISCUSSION TITLE
[Portal engine] Localized fields in block gives error

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#12978] [Q&A / Support] [answered]
I can't open portal product detail page if custom layout for portal contains Block field which contains localized fields.

Steps to reproduce:

Create block field with localized field:
![image](https://user-images.githubusercontent.com/17277953/186647587-2e573845-3195-4157-b866-2fd4a6540694.png)

Fill data in product
![image](https://user-images.githubusercontent.com/17277953/186647663-300d1504-63d1-483a-9015-1a739c5b65e8.png)

Try to open in portal:
![image](https://user-images.githubusercontent.com/17277953/186647846-523d86b3-242c-4c28-ab2e-3a1785791e6d.png)

On my local message is:
![image](https://user-images.githubusercontent.com/17277953/186648026-deafba3a-43e3-47ae-b8a7-753c06782f5a.png)

Are block with localized fields supported in portal ?

Thanks!

**Comment by dvesh3** [comment]
I created a bug issue on portal engine repository. thanks!

**Comment by kingjia90** [comment]
@Maksonis I've tried to reproduce with the same steps on https://portal-demo.pimcore.com/ but it works 🤔 Any details missing? Could you please also retry?

**Reply by Maksonis** [reply]
@kingjia90 did you added it in Portal custom layout ?

![image](https://user-images.githubusercontent.com/17277953/188461453-59643f8e-cd87-4290-b4d4-7c47a0ba777d.png)

For me it's the same:

![image](https://user-images.githubusercontent.com/17277953/188461363-cf5726db-53bb-47ad-943c-062ec08a6c2b.png)

**Reply by kingjia90** [reply]
Ah ok, thank you, now i see it

**Comment by kingjia90** [comment, marked as answer]
@Maksonis A quick update, I've fixed it and opened a PR. It would be likely release in Portal Engine 2.4.2.
